test/spec.dump

```diff
@@ -1,3 +1,29 @@
 {
-"annotations": {"io.container.manager": "libpod"}
+  "ociVersion": "1.0.2-dev",
+  "mounts": [
+    {
+      "destination": "/proc",
+      "type": "proc",
+      "source": "proc",
+      "options": [
+        "nosuid",
+        "noexec",
+        "nodev"
+      ]
+    },
+    {
+      "destination": "/dev",
```

Comment:
You should create an entry that can show the differences of `shortenPath()` during the tests.